### PR TITLE
Create payment class under person

### LINKED ISSUE #46: Add Payment to the model
Editing the add command to include payment
Add a new `Payment` class in the model

### PR DESCRIPTION
**Pull request by Slipperybara**
fixes #46

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Payment.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        this.hasPaid = hasPaid;
+    }
+
+    /**
+     * Constructs an {@code Address}.
+     * Set to false as default
+     */
+    public Payment() {
+        this.hasPaid = false;
+    }
```
Comment (BlazeChron) [inline src/main/java/seedu/address/model/person/Payment.java:28]: hasPaid is final, should it be not final instead? Or should there be a constructor argument?

File: src/main/java/seedu/address/model/person/Payment.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     *
+     * @param hasPaid Boolean for payment status
+     */
+    public Payment(Boolean hasPaid) {
+        requireNonNull(hasPaid);
+        this.hasPaid = hasPaid;
+    }
+
+    /**
+     * Constructs an {@code Address}.
```
Comment (BlazeChron) [inline src/main/java/seedu/address/model/person/Payment.java:23]: javadoc Address

File: src/main/java/seedu/address/model/person/Payment.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+     * Set to false as default
+     */
+    public Payment() {
+        this.hasPaid = false;
+    }
+
+    @Override
+    public String toString() {
+        return hasPaid ? "Paid" : "Not Paid";
```
Comment (BlazeChron) [inline src/main/java/seedu/address/model/person/Payment.java:32]: I forgot, is this ok for coding standard?

**Review by BlazeChron — COMMENTED** [review]
A few nits

File: src/main/java/seedu/address/model/person/Payment.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Student's payment status in the address book.
```
Comment (BlazeChron) [inline src/main/java/seedu/address/model/person/Payment.java:6]: Update to include Guarantees: immutable; like the other classes?

File: src/main/java/seedu/address/model/person/Payment.java (hunk 3 above)
Comment (Slipperybara) [reply src/main/java/seedu/address/model/person/Payment.java:32]: I checked, as for coding standard the ternary operation is ok. 
But I will create a temporary variable to store it and then return the temporary variable.
Thanks for the spot

Comment (Slipperybara) [reply src/main/java/seedu/address/model/person/Payment.java:32]: Nevermind I think currently its the standard :open_mouth:

**Review by amitha2210 — APPROVED** [review]
LGTM

**Pull request merged**